disco_aws_automation/disco_elb.py

```diff
@@ -242,8 +250,20 @@ def get_or_create_elb(self, hostclass, security_groups, subnets, hosted_zone_nam
 
         self.route53.create_record(hosted_zone_name, cname, 'CNAME', elb['DNSName'])
 
-        self._setup_health_check(elb_id, health_check_url, instance_protocol, instance_port, elb_name)
-        self._setup_sticky_cookies(elb_id, elb_ports, sticky_app_cookie, elb_name)
+        for mapping in port_config.port_mappings:
+            self._setup_health_check(
+                elb_id,
+                health_check_url,
```

Comment:
In that case we should define more sensible behavior.  How about, if the user defines a health check URL, create a health check for the first HTTP or HTTPS mapping.  Otherwise create a health check for the first mapping.